Add Sidekiq worker-level success and failure metrics (#1440)
appsignal/appsignal-ruby: .changesets/add-sidekiq-worker-level-success-and-failure-metrics.md

---
bump: minor
type: add
---

> Add Sidekiq worker-level job status metric: `worker_job_count`. This new counter metric's `status` tag will either be `success` or `failure` depending on if it the job encountered an error or not.

Reviewer: ```suggestion
Add Sidekiq worker-level job status metric: `worker_job_count`. This new counter metric's `status` tag will be `processed` for each job that's processed and reports another counter with the `failure` status if the job encountered an error.
```

Small update to reflect the new situation.
